src/web/actions/index.js

```diff
@@ -52,6 +52,8 @@ export const createFlowNode = createAction('FLOWS/FLOW/CREATE')
 export const removeFlowNode = createAction('FLOWS/FLOW/REMOVE')
 export const copyFlowNode = createAction('FLOWS/NODE/COPY')
 export const pasteFlowNode = createAction('FLOWS/NODE/PASTE')
+export const copyFlowNodeElement = createAction('FLOWS/NODE/ELEMENT/COPY')
```

Comment:
Do we have any other (non-node) kind of elements?
Generally I'd recommend keeping action names with 3 levels always, `prefix/entity/action`.
So I'd either go for `FLOWS/ELEMENT/COPY` or `FLOWS/NODE_ELEMENT/COPY` (if there are other kinds of elements)

Reply:
We have "nodes" and "node_elements" so I went with the second one: `FLOWS/NODE_ELEMENT/COPY`